src/microbe_stage/MicrobeStage.cs

```diff
@@ -287,7 +291,8 @@ private void HandlePlayerRespawn()
         var playerSpecies = GameWorld.PlayerSpecies;
 
         // Decrease the population by 20
-        GameWorld.AlterSpeciesPopulation(playerSpecies, -20, "player died", true);
+        GameWorld.AlterSpeciesPopulation(
+            playerSpecies, Constants.PLAYER_DEATH_POPULATION_LOSS, "player died", true);
```

Comment:
Also update the comment, which now may become out of sync with the constant value.